src/briefcase/bootstraps/pygame.py

```diff
@@ -34,6 +35,13 @@ def main():
 
     os.environ["SDL_VIDEO_X11_WMCLASS"] = metadata["Formal-Name"]
 
+    # Set the app's runtime icon
+    pygame.display.set_icon(
+        pygame.image.load(
+            pkg_resources.resource_filename(app_module, f"resources/{app_module}.png")
+        )
+    )
```

Comment:
curious....is there a reason we fetch all of this information at runtime?

When we roll this template out, cookiecutter will have these pieces of information to inject directly in to the source code.

Was there anything preventing doing this originally?

Reply:
What's your precedent for that statement? Toga and PySide don't set an icon; PPB uses a hard-coded path from template data. 

They all use importlib metadata to get the app's *formal* name (Toga's app bootstrap doesn't expose the code, but that's how it's done internally); that's essentially to maintain a single source of metadata truth - the app's formal name can change but the code doesn't need to. However, I think you're right that we know where the resource path is, so we can hard code it. 

The only thing to be wary of is relative paths and CWD resolution. If you specify a file with a relative path, the anchor that is used to resolve the absolute path can vary depending on the platform, and whether you're in `dev` or `run`. PPB fixes this internally; I don't know what PyGame does, but I wouldn't bet money on it doing it in a way that would be consistent. FWIW, retrieving the resource path with importlib is one way to do this... but so is `__file__`.

Reply:
You're right; I was a little too flippant in my assessment that "a lot of work is being done to access something we already know" when it should definitely be dynamically sourced from the metadata.